components/Sribaduga/DaftarPesanan/Detail/index.vue

```diff
@@ -235,6 +237,20 @@ export default {
         this.base64String = encodedString
       }
     },
+    getDataFromCustomerData(data, item) {
+      if (item.customerData) {
+        if (data === 'instance-name') {
+          return item?.customerData.find(
+            (info) => info.questionId === 'instance-name'
+          )?.answer
+        } else if (data === 'name') {
+          return item?.customerData.find((info) => info.questionId === 'name')
+            ?.answer
+        }
+      } else {
+        return '-'
+      }
+    },
```

Comment:
minor:

hello @rizfirman I think we can simplify this function by removing nested if statements like this
```suggestion
      if (!item.customerDate) return '-'
      
      if (data === 'instance-name') {
          return item.customerData.find(
            (info) => info.questionId === 'instance-name'
         )?.answer
      }
      
      if (data === 'name') {
          return item.customerData.find((info) => info.questionId === 'name')
            ?.answer
      }
    },
```

Reply:
yes, that's more simple. i've changed it. thanks!